validator/src/main.rs

```diff
@@ -2360,10 +2340,6 @@ pub fn main() {
 
     let private_rpc = matches.is_present("private_rpc");
     let do_port_check = !matches.is_present("no_port_check");
-    let no_rocksdb_compaction = true;
```

Comment:
@yhchiang-sol i think you can tell all the deleted code here is indded unused because of this hard-coded flag.. ;)